test/e2e/upgrades/ingress.go

```diff
@@ -122,4 +153,81 @@ func (t *IngressUpgradeTest) verify(f *framework.Framework, done <-chan struct{}
 	}
 	By("hitting the Ingress IP " + t.ip)
 	framework.ExpectNoError(framework.PollURL(fmt.Sprintf("https://%v/", t.ip), "", framework.LoadBalancerPollTimeout, t.jig.PollInterval, t.httpClient, false))
+
+	if triggerSync {
```

Comment:
If you're going to keep `triggerSync`, recommend returning early.

```
if !triggerSync {
 return
}
```

Reply:
Removed triggerSync.